.taskcluster.yml

```diff
@@ -59,6 +59,45 @@ tasks:
         owner: taskcluster-internal@mozilla.com
         source: ${event.repository.url}
 
+  - $if: '(tasks_for == "github-pull-request"  && event["action"] in ["opened", "reopened", "synchronize"])'
+    then:
+      taskId: {$eval: as_slugid("windows-tests")}
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '2 hours'}
+      provisionerId: proj-taskcluster
+      workerType: windows2012r2-amd64-ci
+      metadata:
+        name: taskcluster-worker-runner-windows
+        description: windows tests for taskcluster-worker-runner
+        owner: taskcluster-internal@mozilla.com
+        source: ${event.repository.url}
+      payload:
+        maxRunTime: 3600
+        command:
+          - set CGO_ENABLED=0
```

Comment:
Note, you can add static env vars in the `env:` section if you like (although this also works fine).